src/WordTokenizers.jl

```diff
@@ -29,4 +34,84 @@ include("words/TokTok.jl")
 include("set_method_api.jl")
 include("split_api.jl")
 
+include("statistical/unigram.jl")
+
+
+
+const pretrained = Dict{DataType, Vector{String}}()
+function tokenizer_files(::Type{T}) where T<:PretrainedTokenizer 
+    get!(pretrained,T) do
+        String[]
+    end
+end
+function __init__()
+    vectors_albertversion1 = [
```

Comment:
Can we shift the DataDeps registration block into a separate file and inside \__init__ either include file as in  [CorpusLoaders.jl](https://github.com/JuliaText/CorpusLoaders.jl/blob/master/src/CorpusLoaders.jl#L17)
or call function as in [TextAnalysis.jl](https://github.com/JuliaText/TextAnalysis.jl/blob/master/src/TextAnalysis.jl#L133)